client/src/app/(onboarding)/page.tsx

```diff
@@ -1,10 +1,31 @@
 import "./LandingPage.css";
 import { LandingPageHero } from "@/ui/landingpage/LandingPageHero";
+import { Typography } from "@mui/material";
+import Image from "next/image";
 
 export default function Home() {
   return (
     <div>
       <LandingPageHero />
+      {/*Section 2 Block*/}
+      <div className="sec2-container">
+      <div className="sec2-text-box">
+          <Typography variant="h3">Talk to other applicants</Typography>
+
+          <Typography variant="body1">
```

Comment:
Can we make this bigger like in the ones in the designs, including the margin between the two components.
<img width="644" alt="image" src="https://github.com/user-attachments/assets/6d38d91a-3b78-467f-a802-cc01aa967ad3">